CRM/Event/Form/SelfSvcUpdate.php

```diff
@@ -149,19 +166,13 @@ public function preProcess() {
     }
     $details = array_merge($details, $selfServiceDetails);
     $this->assign('details', $details);
-    $this->selfsvcupdateUrl = CRM_Utils_System::url('civicrm/event/selfsvcupdate', "reset=1&id={$this->_participant_id}&id=0");
```

Comment:
None of these 3 properties are used, and furthermore looking at the original commit, I don't believe they ever have been: https://github.com/civicrm/civicrm-core/commit/23cb875c2121a0161e6622071c2ff4eab636e28b